server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInController.java

```diff
@@ -17,7 +17,11 @@
 import io.micronaut.http.annotation.Put;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+
 @Controller("/check-in")
+@Secured(SecurityRule.IS_ANONYMOUS)
```

Comment:
This will be changed to @Secured(SecurityRule.IS_AUTHENTICATED) in issue #80 so that only the authorized users can access the REST controller.

https://docs.micronaut.io/latest/api/io/micronaut/security/rules/SecurityRule.html